stix_edh/bindings/cyber_profile.py

```diff
@@ -884,4 +891,4 @@ def parseString(inString):
     "PolicyType",
     "PublicReleaseType",
     "ResourceDispositionType"
-]
+    ]
```

Comment:
Don't think this indentation is necessary.

Reply:
I didn't give you all of the details about creating this file.  I actually did a diff and brought over the original code where it was different.  So some of these differences are because of that.  I will remove this difference.